Dfe.PrepareConversions/Dfe.PrepareConversions.Tests/TagHelpers/KeyStage4DataTagHelperTests.cs

```diff
@@ -0,0 +1,61 @@
+﻿using Dfe.PrepareConversions.TagHelpers;
+using Dfe.PrepareConversions.Tests.Pages;
+using FluentAssertions;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace Dfe.PrepareConversions.Tests.TagHelpers
+{
+   public class KeyStage4DataTagHelperTests : BaseIntegrationTests
+   {
+      public DateTime _currentDate = DateTime.Now;
+      public KeyStage4DataTagHelperTests(IntegrationTestingWebApplicationFactory factory) : base(factory)
+      {
+      }
+      public static IEnumerable<object[]> ProvisionalDates()
```

Comment:
Its only a small thing but these should probably be private, and if so at the bottom of the file.

Reply:
When using the MemberData attributes from XUnit they require them to be public - but I'll throw them to the bottom of the file!